plugin/src/test/java/io/jenkins/plugins/casc/yaml/MergeStrategyTest.java

```diff
@@ -0,0 +1,30 @@
+package io.jenkins.plugins.casc.yaml;
+
+import hudson.ExtensionList;
+import jenkins.model.Jenkins;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.RestoreSystemProperties;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class MergeStrategyTest {
+
+    @Rule
+    public RestoreSystemProperties properties = new RestoreSystemProperties();
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    @Test
+    public void haveTheDefaultStrategy() {
```

Comment:
is there already a test case for the default strategy?

Reply:
The default strategy already covered by [the exists test cases](https://github.com/jenkinsci/configuration-as-code-plugin/blob/80e556e277b9a9980659eb1131497758e49bf239/plugin/src/test/java/io/jenkins/plugins/casc/ConfigurationAsCodeTest.java#L151).